README.md

```diff
@@ -7,8 +7,8 @@ Questa sezione descrive le specifiche e il flusso di integrazione richiesto per
 ### Servizi mandatori di gestione del recapito
 Per tutti i servizi legati al mondo della stampa e del recapito si faccia riferimento ai seguenti documenti:
 * [gestionePostalizzazione.md](docs/gestionePostalizzazione.md) - descrive l'integrazione con il processo di postalizzazione (stampa ed imbustamento, invio a recapito e successiva rendicontazione)
-* [gestioneErroreRendicontazione.md](docs/gestioneErroreRendicontazione.md) - descrive la modalità di gestione degli errori generati da SEND in fase di validazione delle rendicontazioni ed il processo di recupero \[NUOVASPECIFICA\]
-* [gestioneRendicontazioniSostitutive.md](docs/gestioneRendicontazioniSostitutive.md) - descrive il processo di recupero di rendicontazioni sostitutive alle originali \[NUOVASPECIFICA\]
+* [gestioneErroreRendicontazione.md](docs/gestioneErroreRendicontazione.md) - descrive la modalità di gestione degli errori generati da SEND in fase di validazione delle rendicontazioni ed il processo di recupero \[FASE1\]
```

Comment:
stiamo descrivendo dentro un paragrafo che si chiama servizi mandatori, qualcosa che in realtà ci aspettiamo in una fase successiva